gutenberg.php

```diff
@@ -484,7 +488,11 @@ function gutenberg_replace_default_add_new_button() {
 			display: block;
 			top: 0;
 			margin: -1px 0;
+		<?php if ( is_rtl() ) : ?>
```

Comment:
Just wondering about the indentation here, should it be aligned with the styling (like above) or not? We should probably be consistent anyway.

Reply:
👍 will fix